packages/devtools_server/lib/src/server.dart

```diff
@@ -242,6 +259,24 @@ Future<void> registerLaunchDevToolsService(
   }
 }
 
+final bool _isChromeOS = new File('/dev/.cros_milestone').existsSync();
+
+bool _isAccessibleToChromeOSNativeBrowser(Uri uri) {
+  const tunneledPorts = {
```

Comment:
Add a TODO to switch to a set literal once we can/

Reply:
Done! Though I'm hoping we can delete this method before long (see the TODO further up) :-)